node_modules/gh-orgunit/lib/internal/dao.js

```diff
@@ -760,3 +763,402 @@ var _exportOrgUnitJSON = function(rootId, orgUnits, callback) {
 
     return callback(null, tree);
 };
+
+/* Import */
+
+/**
+ * Import a set of given organisational units, event series and events into an organisational unit
+ *
+ * @param  {MergedTreeData}     mergedTreeData      The merged tree data that contains the tree to import and the data to delete
+ * @param  {Function}           callback            Standard callback function
+ * @param  {Object}             callback.err        An error object, if any
+ */
+var importTree = module.exports.importTree = function(mergedTreeData, callback) {
```

Comment:
What are the expected sizes of these import operations? Just asking because transactions can "overflow". I think there is usually configuration for transaction size limits on the DB side, so it's just something to keep in mind.